config/redirects.yml

```
@@ -330,3 +330,5 @@ redirects:
   "/landing/how-much-do-teachers-get-paid-social": "/landing/how-much-do-teachers-get-paid"
   "/steps-to-become-a-teacher-search": "/steps-to-become-a-teacher"
   "/landing/home": "/"
+  "/train-to-be-a-teacher/if-you-have-a-degree-search": "/landing/train-to-teach-if-you-have-a-degree"
```

```suggestion
  "/train-to-be-a-teacher/if-you-have-a-degree-search": "/train-to-be-a-teacher/if-you-have-a-degree"
```

We just want to direct any traffic that was going to the copy back to the original page @ethax-ross, rather than to the new landing page :-)

Other changes all look fine so happy to approve/merge tomorrow (Weds)

---

Whoops, updated now